### ISSUE TITLE
Emphasise documentation and tutorials

### ISSUE THREAD

**Issue by hyanwong** [petrelharp/progen-2023#2]
I think it is a strength of tskit that we have so much documentation and learning resources online. This should definitely be highlighted (with links?) somewhere on the poster, as it should encourage others to try it out.

**Comment by jeromekelleher** [comment]
I think a big ``https://tskit.dev`` somewhere prominent would be good.

Perhaps QR codes?

**Comment by gtsambos** [comment]
Yes, we thought qr codes would be good too. Perhaps to a directory with
notebooks for all the panels, or to the tutorials page

**Comment by hyanwong** [comment]
See https://github.com/petrelharp/progen-2023/issues/1#issuecomment-1452559000 for a QR code from Kevin.
